src/Common/Telemetry/Telemetry.php

```diff
@@ -123,7 +130,7 @@ public function boot(): void {
 			return;
 		}
 
-		$telemetry_server   = ! defined('TELEMETRY_SERVER') ? 'https://telemetry.stellarwp.com/api/v1': TELEMETRY_SERVER;
+		$telemetry_server   = ! defined('STELLARWP_TELEMETRY_SERVER') ? 'https://telemetry.stellarwp.com/api/v1': STELLARWP_TELEMETRY_SERVER;
```

Comment:
This is the cross-product constant we agreed upon. Was going to update it in k9 anyway.